src/odemis/gui/cont/acquisition/sparc_acq.py

```diff
@@ -248,6 +284,9 @@ def update_acquisition_time(self):
             txt = u"Estimated time is {}."
             txt = txt.format(units.readable_time(acq_time))
 
+        # check if the acquire button needs enabling or disabling
+        self.check_acquire_button()
```

Comment:
Are there any other callers? If not, just move that function here (it's a single line).